### PR TITLE
List room type

### PR DESCRIPTION
**Pull request by at-tringuyen3**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Task ID: TI-9

## Reference Isuees
#9

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: app/Http/Controllers/RoomTypeController.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\RoomType;
+
+class RoomTypeController extends Controller
+{
+
+    protected $roomTypes;
```
Comment (at-tuannguyen3) [inline app/Http/Controllers/RoomTypeController.php]: sửa lại cho anh chổ này với $roomType;

File: app/Http/Controllers/RoomTypeController.php [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $roomType = $this->roomTypes->getRoomTypes();
```
Comment (at-tuannguyen3) [inline app/Http/Controllers/RoomTypeController.php]: này sửa lại là roomTypes nhé

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment! tks you

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**